helm/prometheus-operator-app/README.md

```diff
@@ -77,16 +71,23 @@ kubectl delete crd thanosrulers.monitoring.coreos.com
 ## Upgrading Chart
 
 ```console
-# Helm 3 or 2
+# Helm
 $ helm upgrade [RELEASE_NAME] prometheus-community/kube-prometheus-stack
 ```
 
+With Helm v3, CRDs created by this chart are not updated by default and should be manually updated.
+Consult also the [Helm Documentation on CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions).
```

Comment:
This is not true, thanks to @glitchcrab and the work from https://github.com/giantswarm/prometheus-operator-app/pull/63.

CRDs should now always be updated.

Reply:
This readme is synced directly from the repo. I am wondering if your sync mechanism could be proposed upstream